client/src/components/Login/LoginInterface.js

```diff
@@ -53,7 +53,34 @@ export class LoginInterface extends Component {
       
     if(rules.required){
       isValid = value.trim() !== '' && isValid;
-    } 
+    }
+
+    if (rules.minLength) {
+      isValid = value.length >= rules.minLength && isValid;
+      console.log("minlength: " + isValid);
+      isValid === false ? this.setError("password", "Password must be longer than 5 characters") : this.removeError("email");
+    }
+
+    if (rules.isEmail) {
```

Comment:
I mean this down here

Reply:
Oh I see, it looks like these rules are here but not being used. Should be removed then.